grammar.js

```diff
@@ -5,8 +5,12 @@ module.exports = grammar({
     /\s/
   ],
 
+  supertypes: $ => [
+    $._value
+  ],
+
   rules: {
-    value: $ => $._value,
+    document: $ => $._value,
```

Comment:
Renamed the top-level rule to `document` so that we can have `_value` in the `supertypes` without collision.